source/_patterns/00-atoms/inline/date/date.twig

```diff
@@ -0,0 +1,11 @@
+{%- set attributes = attributes|default({})|merge({datetime: date|date("Y-m-d")}) -%}
+
+{% spaceless %}
+
+  <time {%- include 'atoms-attributes' %}>
+
+    <bdo>{{ date|localizeddate(format|default('long'), 'none', attributes.lang|default(lang ?? null)) }}</bdo>
```

Comment:
Part of me is now wondering whether the pattern should actually just take a string, so the equivalent of this would happen in Browser. /cc @davidcmoulton

Reply:
eLife has entries like ‘Updated Mar 29, 2019’ which is should be thought of as one piece of text.

Reply:
Happy with that, I don't have a strong opinion either way.

Reply:
Will take the simple option and reduce this to a `time` pattern that requires input. We can see how it ends up being used as create something more specialised if needed.